### DISCUSSION TITLE
Extend `$header` modifier

### DISCUSSION THREAD

**Discussion by piquark6046** [uBlockOrigin/uBlock-issues#2254] [Ideas]
What about extending matching ability of  `$header` modifier to HTTP request header?
 - https://github.com/gorhill/uBlock/commit/bde3164eb445a4e74acca303ec9fa07f82ba1b1c
 - https://listauthorschat.slack.com/archives/C010N14G4TF/p1662830803562149

**Comment by krystian3w** [comment]
related ask: https://github.com/uBlockOrigin/uBlock-issues/issues/2156
